### DISCUSSION TITLE
【Mod錯誤】如截图，无法打开游戏

### DISCUSSION THREAD

**Discussion by Bailan99** [FrozenFish259/Goose_Goose_Duck_Hack#336] [Q&A 疑难解答区]
![123](https://user-images.githubusercontent.com/121304102/221382124-bddee2b9-ca7d-49f8-863e-53484ec744a3.jpg)

还有请问QQ群密码是多少，想群里问，但是群密码不知道

（未安装mod时，辅助可以正常打开，游戏也正常打开，安装后出现截图的弹窗）

**Comment by Bailan99** [comment]
目录含有中文会有上面的错误，我尝试改为英文目录就正常安装了，但是进入游戏时出现了新的错误提示，每次都卡在这里


![image](https://user-images.githubusercontent.com/121304102/221383100-bf91ce1f-10ea-459f-8f89-c0e958ce91ea.png)

**Comment by Bailan99** [comment]
錯誤截圖補充，試了剛更新的  1.20版本mode.dll   還是一樣的錯誤
 
![image](https://user-images.githubusercontent.com/121304102/221383392-9b3c323f-65e3-4c54-8ba1-2ff9a468a5ea.png)
![UMDWQ72VB7((9KL44L(U%3](https://user-images.githubusercontent.com/121304102/221383399-dd901025-43a8-4d1a-8f4a-daf584e5de4a.png)

**Reply by FrozenFish259** [reply]
框架没装好吧
